app/services/idv/steps/doc_auth_base_step.rb

```diff
@@ -130,6 +130,10 @@ def verify_step_document_capture_session_uuid_key
         :idv_verify_step_document_capture_session_uuid
       end
 
+      def service_provider_device_profiling_enabled?
+        ServiceProvider.find_by(issuer: sp_session[:issuer])&.device_profiling_enabled
```

Comment:
Should we skip the query if `issuer` is `nil`? Also, what should be the default if there is no issuer associated? 

```suggestion
        return false if sp_session[:issuer].blank?
        ServiceProvider.find_by(issuer: sp_session[:issuer])&.device_profiling_enabled
```

Reply:
That is a great question.  There would be no way to turn on device profiling for the no issuer (even though that is the default for all sps) unless perhaps we start using the blank sp for the no sp case.  I'm not sure we care about this now.  If we do I can add another global app setting for the blank sp case.

Reply:
I think the current plan is to skip device profiling for the no-issuer case, but that assumption may change in the future (if we do a full rollout) so it seems worthwhile adding a feature flag for that now

Reply:
On it...